saviSimulation/savi/jason_processing/SAVI_Controller.java

```diff
@@ -0,0 +1,73 @@
+package savi.jason_processing;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.util.Properties;
+
+import processing.core.PApplet;
+import savi.commandStation.CommandStationCore;
+
+public class SAVI_Controller {
+		
+	
+	public static void main(String [] args) {
+		
+			
+		FileInputStream in = null;
+		Properties modelProps = new Properties();
+		
+		/*** LOAD SIM PARAMETERS ****/
+		try {
+			String filePath = new File("").getAbsolutePath();
+			filePath = filePath + "/config.cfg";
+			System.out.println(filePath);
+			File inFile = new File(filePath);
+			in = new FileInputStream(inFile);
+			modelProps.load(in);
```

Comment:
I really like the idea of having a properties object. I think this can help clean up the constructors throughout the project.